Lib/uuid.py

```diff
@@ -159,15 +159,18 @@ def __init__(self, hex=None, bytes=None, bytes_le=None, fields=None,
                 raise ValueError('badly formed hexadecimal UUID string')
             int = int_(hex, 16)
         if bytes_le is not None:
+            if not isinstance(bytes_le, (bytes_, bytearray)):
```

Comment:
Add `int`.

Reply:
I don't understand why bytes_le should accept an int? Do you have an example please?

Reply:
It should not! But accepts. Add a test for `bytes_le=16`. :wink: